src/app/modules/members/components/members-list/members-list-filter-bar/members-list-filter-bar.component.ts

```diff
@@ -0,0 +1,46 @@
+import { Component, EventEmitter, Output } from '@angular/core';
+import { Router } from '@angular/router';
+import { FilterValues } from './filter-values.model';
+import { InputValues } from './input-values.model';
+
+@Component({
+  selector: 'vc-members-list-filter-bar',
+  templateUrl: './members-list-filter-bar.component.html',
+  styleUrls: [
+    './members-list-filter-bar.component.scss',
+  ],
+})
+
+export class MembersListFilterBarComponent {
+  searchFilterValue = '';
+
+  selectInputValue = 'all';
+
+  @Output() filterChange = new EventEmitter<FilterValues>();
+
+  constructor(private readonly router: Router) {}
+
+  onValueChanges({ value, emitterElement } : InputValues): void {
+    switch (emitterElement) {
+    case 'input':
```

Comment:
You probably have a typo here: searchFilterValue is for 'select' and selectInputValue for 'input'

Reply:
I'm not sure about having this outputs in this story at all. When we need direct access to HTML Elements in angular it's always is sign something going wrong. Switching by input type is not something good too. Let's leave them empty until next stories and have a discussion about how it should working.